pkg/controllers/provisioning/scheduler.go

```diff
@@ -102,7 +102,7 @@ func (s *Scheduler) Schedule(ctx context.Context, pod *v1.Pod) error {
 	return nil
 }
 
-func isUnschedulable(p *v1.Pod) error {
+func isSchedulable(p *v1.Pod) error {
```

Comment:
@JacobGabrielson and I went back and forth on this naming, since there was a similar thing that bit us recently. I think the weird bit is that since the function returns an error, but it seems like it should return a boolean, and that if the boolean is true, then it's the happy case. Instead, you need to write isSchedulable != nil, which is really confusing.

We could make the functions all oriented around the validate concept, which semantically returns an error.

e.g.

```
validateSchedulable(pod) // potentially makes more sense as provisionable, since it technically is not schedulable by the scheduler.
```

Reply:
Alternatively, it doesn't look like we're using these error messages anymore, so you could just make this a good old fashioned boolean. That maybe results in the simplest outcome `if isProvisionable(pod) { }`.